lib/plugins/aws/package/compile/events/websockets/index.js

```diff
@@ -51,9 +51,45 @@ class AwsCompileWebsockets {
       },
     };
 
-    // TODO: Complete schema, see https://github.com/serverless/serverless/issues/8019
     this.serverless.configSchemaHandler.defineFunctionEvent('aws', 'websocket', {
-      anyOf: [{ type: 'string' }, { type: 'object' }],
+      anyOf: [
+        { type: 'string' },
+        {
+          type: 'object',
+          properties: {
+            route: { type: 'string' },
+            routeResponseSelectionExpression: {
+              type: 'string',
+              regexp: /^(?:\$default)$/.toString(),
+            },
+            authorizer: {
+              anyOf: [
+                { type: 'string' },
+                {
+                  type: 'object',
+                  properties: {
+                    name: { type: 'string' },
```

Comment:
We coudld use `functionName` definition as proposed in previous comment

Reply:
done!